.azure-pipelines.yml

```diff
@@ -47,13 +47,13 @@ jobs:
         env:
           GH_TOKEN: $(LENS_IDE_GH_TOKEN)
         displayName: Customize config
-      
+
       - script: make node_modules
-        displayName: Install dependencies
-      
+        displayName: Install dependencies=
```

Comment:
what does the `=` do?

Reply:
oops that was accidental when reverting the previous change.